sdk/eventhub/Azure.ResourceManager.EventHubs/tests/Tests/ScenarioTests.ConsumerGroupsTests.CRUD.cs

```diff
@@ -28,7 +28,7 @@ public async Task ConsumerGroupsCreateGetUpdateDelete()
                     {
                         Tier = SkuTier.Standard,
                     },
-                    Tags = new Dictionary<string, string>()
```

Comment:
@pakrym , when running this code, NullReferenceException is throw after removing `new Dictionary<string, string>()` here. It seems that `Tags` is not initialized with `new Dictionary<>` in TrackedResource.cs, could you take a look?

Reply:
Are you sure the exception happens on this line? 

Tags collection is initialized in ctor https://github.com/Azure/azure-sdk-for-net/pull/13585/files#diff-ee5ab4c1618b272e8775ca1a33b97a56R19